fcrepo-kernel-api/src/main/java/org/fcrepo/kernel/api/identifiers/FedoraID.java

```diff
@@ -156,6 +188,88 @@ public String getMementoString() {
         return mementoDatetimeStr;
     }
 
+    /**
+     * Create a new FedoraID using the resource ID as the base
+     * @param addition One or more additional strings.
+     * @return new FedoraID for the new identifier.
+     */
+    public FedoraID addToResourceId(final String... addition) {
```

Comment:
I think I understand what you are saying, but I am unclear on what you are suggesting it looks like.

Reply:
This partly depends on what you think about another suggestion, further down:
https://github.com/fcrepo4/fcrepo4/pull/1652#discussion_r404279803

But if we end up keeping fcr:metadata in the fullId, a call to `resolveId(String... additions)` would inspect the additions. If one of them was fcr:metadata, I believe we would want it to call `return FedoraID.create(this.getResourceId(), addition);`, otherwise we call `return FedoraID.create(this.getFullId(), addition);`